test/DGmethods/advection_diffusion/advection_sphere.jl

```diff
@@ -0,0 +1,462 @@
+    norm(Q - Qe)            = %.16e
+    norm(Q - Qe) / norm(Qe) = %.16e
+    """ Δmass engf engf / eng0 engf - eng0 errf errf / engfe
+    return errf, Δmass
+end
+
+using Test
+let
+    CLIMA.init()
+    ArrayType = CLIMA.array_type()
+
+    mpicomm = MPI.COMM_WORLD
+
+    polynomialorder = 4
+    base_num_elem = 2
+
+    max_cfl = Dict(
+        LSRK144NiegemannDiehlBusch => 5.0,
+        SSPRK33ShuOsher => 1.0,
+        SSPRK34SpiteriRuuth => 1.5,
+    )
+
+    expected_result = Dict()
+
+    expected_result[SolidBodyRotation, LSRK144NiegemannDiehlBusch, 1] =
+        1.3199024557832748e-01
+    expected_result[SolidBodyRotation, LSRK144NiegemannDiehlBusch, 2] =
+        1.9868931633120656e-02
+    expected_result[SolidBodyRotation, LSRK144NiegemannDiehlBusch, 3] =
+        1.4052110916915061e-03
+    expected_result[SolidBodyRotation, LSRK144NiegemannDiehlBusch, 4] =
+        9.0193766298676310e-05
+
+    expected_result[SolidBodyRotation, SSPRK33ShuOsher, 1] =
+        1.1055145388897809e-01
+    expected_result[SolidBodyRotation, SSPRK33ShuOsher, 2] =
+        1.5510740467668628e-02
+    expected_result[SolidBodyRotation, SSPRK33ShuOsher, 3] =
+        1.8629481690361454e-03
+    expected_result[SolidBodyRotation, SSPRK33ShuOsher, 4] =
+        2.3567040048588889e-04
+
+    expected_result[SolidBodyRotation, SSPRK34SpiteriRuuth, 1] =
+        1.2641959922001456e-01
+    expected_result[SolidBodyRotation, SSPRK34SpiteriRuuth, 2] =
+        2.2780375948714751e-02
+    expected_result[SolidBodyRotation, SSPRK34SpiteriRuuth, 3] =
+        3.1274951764826459e-03
+    expected_result[SolidBodyRotation, SSPRK34SpiteriRuuth, 4] =
+        3.9734060514021565e-04
+
+    expected_result[ReversingDeformationalFlow, LSRK144NiegemannDiehlBusch, 1] =
+        5.5387951598735408e-01
+    expected_result[ReversingDeformationalFlow, LSRK144NiegemannDiehlBusch, 2] =
+        3.7610388138383732e-01
+    expected_result[ReversingDeformationalFlow, LSRK144NiegemannDiehlBusch, 3] =
+        1.7823508719111605e-01
+    expected_result[ReversingDeformationalFlow, LSRK144NiegemannDiehlBusch, 4] =
+        3.8639493470255713e-02
+
+    expected_result[ReversingDeformationalFlow, SSPRK33ShuOsher, 1] =
+        5.5353962032596349e-01
+    expected_result[ReversingDeformationalFlow, SSPRK33ShuOsher, 2] =
+        3.7645487928038762e-01
+    expected_result[ReversingDeformationalFlow, SSPRK33ShuOsher, 3] =
+        1.7823263736245307e-01
+    expected_result[ReversingDeformationalFlow, SSPRK33ShuOsher, 4] =
+        3.8605903366230925e-02
+
+    expected_result[ReversingDeformationalFlow, SSPRK34SpiteriRuuth, 1] =
+        5.5404045660832824e-01
+    expected_result[ReversingDeformationalFlow, SSPRK34SpiteriRuuth, 2] =
+        3.7788858038003154e-01
+    expected_result[ReversingDeformationalFlow, SSPRK34SpiteriRuuth, 3] =
+        1.8007113931230376e-01
+    expected_result[ReversingDeformationalFlow, SSPRK34SpiteriRuuth, 4] =
+        3.9941331660544775e-02
```

Comment:
These errors seem a bit high to me. If you refine more are they better?

Reply:
Yes, they get better with refinement and the rate of convergence approaches the correct value. It is just really bad on coarse meshes, but this matches my previous experience with this test.

Reply:
Sounds good. Just wanted to check.